scalableminds/webknossos
# Add additional coordinates to segment index

### Steps to test:
- Open nd dataset and create some segments
- Open segment context menu in viewport and see BB and volume info
- In the segments tab, click on a group and see the segment statistics. Also test CSV export.
- Make sure everything is working for 3D datasets, such as export
- Make sure ad-hoc mesh rendering and its download is still working

### TODOs:
	- [x]  Supply additional coordinates on segment statistics routes
	- [x] Use the new POST        /volume/:tracingId/segmentIndex/:segmentId  route (instead of GET), which also supports additional coordinates
	- [x] take care of and remove TODO comments in `frontend/javascripts/oxalis/view/context_menu.tsx`, ll. 1166 and 1308
	- [x] fix l. 1098 in https://github.com/scalableminds/webknossos/pull/7394/files/e971675c54c92a55e8c37de12835c1efa4435b76#r1404463304
	- [x] remove condition in segment_mesh_saga > loadAdHocMesh: `usePositionsFromSegmentStats`
	- [x] test mesh download (currently blocked by local CSP)

### Issues:
- fixes #7276

------
(Please delete unneeded items, merge only when none are left open)
